src/class-config.php

```diff
@@ -446,7 +446,7 @@ protected function register_graphql_field( $type_name, $field_name, $config ) {
 				$field_config['type'] = 'String';
 				break;
 			case 'range':
-				$field_config['type'] = 'Integer';
+				$field_config['type'] = 'String';
```

Comment:
This should probably be `Float` to support decimals.

Reply:
Of course! I was in such a rush that I didn't think of that at the time. That's updated now!